source/geometry/FloatPoint2D.ooc

```diff
@@ -21,8 +21,10 @@ use ooc-base
 
 FloatPoint2D: cover {
 	x, y: Float
```

Comment:
Another interesting style issue. `x, y: Float`, or
```ooc
x: Float
y: Float
```
This way is obviously better here, but is it always?